### ISSUE TITLE
Anyone can tell me about the 'loss_cls' how to caculate?

### ISSUE THREAD

**Issue by BCWang93** [eriklindernoren/PyTorch-YOLOv3#134]
In this code ,I think the tcls should like this
'            tcls[b, best_n, gj, gi, target_label] = target_label'
and,the loss_clss should like this:'loss_cls=ce(pred_cls[mask==1],tcls[mask==1])'
I don't understand why use 'torch.argmax(tcls[mask==1],1)',Anyone can help me?Thanks!

**Comment by eriklindernoren** [comment]
`'torch.argmax(tcls[mask==1],1)` was used because `torch.nn.CrossEntropy` loss expects a numerical ground truth label, e.g. `3` for the output `[0, 0, 1]` if there are three classes. You can read more in the PyTorch docs.
